src/Backend/Modules/MediaGalleries/Actions/MediaGalleryEdit.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Actions;
+
+use Backend\Core\Engine\Base\ActionEdit as BackendBaseActionEdit;
+use Backend\Core\Engine\Model;
+use Backend\Modules\MediaGalleries\Domain\MediaGallery\Command\UpdateMediaGallery;
+use Backend\Modules\MediaGalleries\Domain\MediaGallery\MediaGallery;
+use Backend\Modules\MediaGalleries\Domain\MediaGallery\MediaGalleryType;
+
+/**
+ * This is the class to Edit a MediaGallery
+ */
+class MediaGalleryEdit extends BackendBaseActionEdit
+{
+    /**
+     * Execute the action
+     *
+     * @return void
+     */
+    public function execute()
+    {
+        parent::execute();
+
+        /** @var MediaGallery $mediaGallery */
+        $mediaGallery = $this->getMediaGallery();
+
+        $form = $this->createForm(
+            new MediaGalleryType(
```

Comment:
fork 5 won't allow this way of creating forms anymore

The UpdateMediaGallery::class is an option and should be passed to the form in the option array

you will only be able to pass the class name (if there are real constructor options you need to add them in the service definition)